examples/src/bin/builders.rs

```diff
@@ -0,0 +1,48 @@
+//! # Builders Sample
+//!
+//! This sample demonstrates how to create a widget using the builders.
+//! These allow to set construct-only properties and other construct
+//! properties when creating the widget.
+
+use gio::prelude::*;
+use gtk::prelude::*;
+
+use gtk::{Align, Application, ApplicationWindowBuilder, ButtonBuilder};
+
+use std::env::args;
+
+fn build_ui(application: &Application) {
+    let window = ApplicationWindowBuilder::new()
+        .application(application)
+        .title("First GTK+ Program")
```

Comment:
```suggestion
        .title("First GTK Program")
```
GTK+ was renamed to GTK few months ago